store/tikv/coprocessor.go

```diff
@@ -86,9 +86,9 @@ func (c *CopClient) Send(ctx context.Context, req *kv.Request, vars *kv.Variable
 		// Make sure that there is at least one worker.
 		it.concurrency = 1
 	}
-	if it.req.KeepOrder {
-		it.sendRate = newRateLimit(2 * it.concurrency)
-	} else {
+
+	it.sendRate = newRateLimit(it.concurrency)
+	if !it.req.KeepOrder {
```

Comment:
If keepOrder is true, the sendRate should be 2 * it.concurrency

Reply:
updated.